components/ProblematicComponent.tsx

```
@@ -0,0 +1,132 @@
+        <p>{item.description}</p>
+      </div>
+    ))
+  }
+  
+  // Accessibility issues
+  return (
+    <div>
+      <h1>Problematic Component</h1>
+      
+      {/* Missing alt text */}
+      <img src="/image.jpg" />
```

Biome (1.9.4)

[error] Provide a text alternative through the alt, aria-label or aria-labelledby attribute

Meaningful alternative text on elements helps users relying on screen readers to understand content's purpose within a page.
If the content is decorative, redundant, or obscured, consider hiding it from assistive technologies with the aria-hidden attribute.

(lint/a11y/useAltText)